tests/backup/backup_resiliency_test.go

```diff
@@ -750,41 +750,24 @@ var _ = Describe("{CancelAllRunningBackupJobs}", func() {
 				}
 				_, err = task.DoRetryWithTimeout(backupJobCancelStatus, maxWaitPeriodForBackupJobCancellation*time.Minute, backupJobCancellationRetryTime*time.Second)
 				if err != nil {
-					adminBackups, error := GetAllBackupsAdmin()
+					adminBackups, error1 := GetAllBackupsAdmin()
 					log.Infof("The list of backups after backup cancellation and wait of 10 minutes is %v", adminBackups)
-					log.FailOnError(error, "Getting the list of backups after backup cancellation and wait of 10 minutes")
+					log.FailOnError(error1, "Getting the list of backups after backup cancellation and wait of 10 minutes")
 				}
-				dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying backup jobs cancellation while backup is in progress"))
 			}
 			log.Infof("All the backups created by this testcase is deleted after backup cancellation")
 		})
 	})
 
 	JustAfterEach(func() {
 		defer EndPxBackupTorpedoTest(contexts)
-		var wg sync.WaitGroup
 		ctx, err := backup.GetAdminCtxFromSecret()
 		log.FailOnError(err, "Fetching px-central-admin ctx")
 		log.Infof("Deleting the deployed applications")
 		opts := make(map[string]bool)
 		opts[SkipClusterScopedObjects] = true
 		ValidateAndDestroy(contexts, opts)
 		log.InfoD("Deleting the remaining backups in case of failure")
```

Comment:
log.InfoD("Deleting the remaining backups in case of failure") might be unnecessary.

Reply:
Thanks!